hazelcast/src/main/java/com/hazelcast/config/HotRestartPersistenceConfig.java

```diff
@@ -90,6 +98,21 @@ public HotRestartPersistenceConfig setBaseDir(File baseDir) {
     }
 
     /**
+     * Gets the configured number of Hot Restart store instance to create for one Hazelcast instance.
+     */
+    public int getParallelism() {
+        return parallelism;
+    }
+
+    /**
+     * Sets the number of Hot Restart store instances to create for one Hazelcast instance.
+     */
+    public HotRestartPersistenceConfig setParallelism(int parallelism) {
```

Comment:
In the config we do a fail fast. So a very basic check if someone doesn't provide a negative value (I don't know if that is allowed or not). So a quick detection of basic input failure.